Tests/BluetoothTests/DarwinTests.swift

```diff
@@ -357,6 +357,240 @@ final class DarwinTests: XCTestCase {
+            0x2782: "ngstrm",
+            0x2783: "nauticalMile",
+            0x2784: "barn",
+            0x2785: "velocityKnot",
+            0x2786: "neper",
+            0x2787: "bel",
+            0x27A0: "yard",
+            0x27A1: "parsec",
+            0x27A2: "inch",
+            0x27A3: "foot",
+            0x27A4: "mile",
+            0x27A5: "pressurePoundForce",
+            0x27A6: "kilometrePerHour",
+            0x27A7: "milePerHour",
+            0x27A8: "revolutionPerMinute",
+            0x27A9: "gramCalorie",
+            0x27AA: "kilogramCalorie",
+            0x27AB: "kilowattHour",
+            0x27AC: "degreeFahrenheit",
+            0x27AD: "percentage",
+            0x27AE: "perMille",
+            0x27AF: "beatsPerMinute",
+            0x27B0: "ampereHours",
+            0x27B1: "milligramPerDecilitre",
+            0x27B2: "millimolePerLitre",
+            0x27B3: "year",
+            0x27B4: "month",
+            0x27B5: "concentration",
+            0x27B6: "irrandiance",
+            0x27B7: "millilitre",
+            0x27B8: "pound",
+            0x27B9: "metabolicEquivalent",
+            0x27BA: "step",
+            0x27BC: "stroke",
+            0x27BD: "pace",
+            0x27BE: "luminousEfficacy",
+            0x27BF: "luminousEnergy",
+            0x27C0: "luminousExposure",
+            0x27C1: "massFlow",
+            0x27C2: "volumeFlow"
+        ]
+        
+        var generatedCode = ""
+        
+        var memberNameCache = [UInt16: String]()
+        
+        func 🖨(_ text: String) {
+            
+            generatedCode += text + "\n"
+        }
+        
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateStyle = .short
+        dateFormatter.timeStyle = .none
+        
+        let fileDate = dateFormatter.string(from: Date())
+        
+        🖨("//")
+        🖨("//  UnitIdentifierExtension.swift")
+        🖨("//  Bluetooth")
+        🖨("//")
+        🖨("//  Generated by Carlos Duclos on \(fileDate).")
+        🖨("//")
+        🖨("")
+        🖨("public extension UnitIdentifier {")
+        🖨("")
+        
+        for (identifier, unit) in units {
+            
+            var memberName = unitsMethodNames[identifier]!
+            
+            // prevent duplicate entries
+            var duplicateNumber = 1
+            while memberNameCache.values.contains(memberName) {
+                
+                duplicateNumber += 1
+                memberName = memberName + "\(duplicateNumber)"
+            }
+            
+            let hexValue = "0x\(identifier.toHexadecimal())"
+            let comment = unit.name + " " + "(`\(hexValue)`)"
+            
+            🖨("    /// " + comment)
+            🖨("    static var " + memberName + ": UnitIdentifier {")
+            🖨("        return UnitIdentifier(rawValue: \(hexValue))")
+            🖨("    }")
+            🖨("")
+            
+            memberNameCache[identifier] = memberName
```

Comment:
`memberNameCache == unitsMethodNames` so `memberNameCache` should be removed

Reply:
@colemancda I'm using memberNameCache to validate duplicates, so memberNameCache != unitsMethodNames xD